lib/devise_two_factor/models/two_factor_authenticatable.rb

```diff
@@ -45,11 +45,6 @@ def current_otp
         otp.at(Time.now)
       end
 
-      # ROTP's TOTP#timecode is private, so we duplicate it here
-      def current_otp_timestep
```

Comment:
Self-note: This changes the API for TwoFactorAuthenticatable models and so we'll need to do a major version bump.